extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -278,19 +278,30 @@ export class AmpA4A extends AMP.BaseElement {
           return Promise.reject(cancellation());
         }
         if (fetchResponse && fetchResponse.arrayBuffer) {
-          return fetchResponse.arrayBuffer().then(
-            bytes => {
-              if (promiseId != this.promiseId_) {
-                return Promise.reject(cancellation());
-              }
-              return this.validateAdResponse_(fetchResponse, bytes)
-                .then(valid => {
+          return fetchResponse.arrayBuffer().then(bytes => {
+            if (promiseId != this.promiseId_) {
+              return Promise.reject(cancellation());
```

Comment:
Ended up doing this after all.  Moving to a throw let me make it a little convenience method as well, which made the code more concise.